Jenkinsfile

```
@@ -0,0 +1,59 @@
+def secrets = [
+    [path: params.VAULT_PATH_SVC_ACCOUNT_EPHEMERAL, engineVersion: 1, secretValues: [
+        [envVar: 'OC_LOGIN_TOKEN_DEV', vaultKey: 'oc-login-token-dev'],
+        [envVar: 'OC_LOGIN_SERVER_DEV', vaultKey: 'oc-login-server-dev']]],
+    [path: params.VAULT_PATH_QUAY_PUSH, engineVersion: 1, secretValues: [
+        [envVar: 'QUAY_USER', vaultKey: 'user'],
+        [envVar: 'QUAY_TOKEN', vaultKey: 'token']]],
+    [path: params.VAULT_PATH_RHR_PULL, engineVersion: 1, secretValues: [
+        [envVar: 'RH_REGISTRY_USER', vaultKey: 'user'],
+        [envVar: 'RH_REGISTRY_TOKEN', vaultKey: 'token']]]
+]
+
+def configuration = [vaultUrl: params.VAULT_ADDRESS, vaultCredentialId: params.VAULT_CREDS_ID, engineVersion: 1]
+
+pipeline {
+    agent { label 'insights' }
+    options {
+        timestamps()
+    }
+    environment {
+        // --------------------------------------------
+        // Options that must be configured by app owner
+        // --------------------------------------------
+        APP_NAME="ingress"
+        COMPONENT_NAME="storage-broker"
+        IMAGE="quay.io/cloudservices/insights-storage-broker"  
+
+        IQE_PLUGINS="e2e"
```

@tahmidefaz @justinorringer let's remove this here too
```suggestion
        IQE_PLUGINS=""
```